test/playwright/test_ux.py

```diff
@@ -120,7 +120,7 @@ def _create_new_environment(page, screenshot=False):
 
 
 def _existing_environment_interactions(
-    page, env_name, time_to_build_env=3 * 60 * 1000, screenshot=False
+    page, env_name, time_to_build_env=5 * 60 * 1000, screenshot=False
```

Comment:
I increased this value because I also saw "Timeout 180000ms exceeded" in the GitHub Action logs